Makefile

```diff
@@ -98,6 +99,17 @@ test: ## Test code using go test
 	@echo "### Testing code"
 	GOOS=$(GOOS) go test -mod vendor -a ./... -coverpkg=./... -coverprofile cover.out
 
+.PHONY: e2e
+e2e: IMAGE=localhost/netobserv-cli:test
+e2e: PULL_POLICY=Never
```

Comment:
:point_up: We must use `Never` for kind cluster; else the pod fall into `ErrImagePull`